deployments/kubernetes/chart/reloader/Chart.yaml

```diff
@@ -1,9 +1,10 @@
+---
 # Generated from deployments/kubernetes/templates/chart/Chart.yaml.tmpl
 
 apiVersion: v1
 name: reloader
 description: Reloader chart that runs on kubernetes
-version: v0.0.31
+version: v0.0.31-1
```

Comment:
@hakamadare kindly remove this, it will update automatically once this PR is merged.